### ISSUE TITLE
Some Issues...

### ISSUE THREAD

**Issue by joe2209** [uhm-coe/insert-pages#20]
Firstly, the retina icon (to insert shortcode) doesn't display in any of my editors, including the Gutenburg editor. I have to manually type in the code.

Secondly, the pages I include using the shortcode seem to be wrapped in PRE and CODE HTML tags, yet I don't see these tags anywhere in the plugins insert-pages.php file. I need to be able to remove these tags from the template because it's causing it to look awful and causes my browsers horizontal scrollbar to display (since the paragraphs within the inserted page are now overscrolling off my screen).

The alternative to removing these tags would be to use a custom template within the shortcode, but I have no idea what I need to put in this custom template.

So, in conclusion:
1). Can we get the icon to show up in my Gutenburg editor?
2). Can you give me a sample template to use for a custom template, or possibly just tell me how to remove the PRE and CODE tags from within the default template?

Thanks.

**Comment by figureone** [comment]
I can't reproduce any of your issues; I suspect another plugin or something in your theme causing problems. You can try turning off plugins one at a time to narrow it down, or switching to one of the twenty* themes that comes with WordPress.

<img width="681" alt="Screen Shot 2019-10-17 at 8 11 51 AM" src="https://user-images.githubusercontent.com/125274/67035843-2a8b5800-f0b6-11e9-98d8-a652f9788501.png">
<img width="951" alt="Screen Shot 2019-10-17 at 8 13 01 AM" src="https://user-images.githubusercontent.com/125274/67035844-2a8b5800-f0b6-11e9-9673-13f59b1b4ff0.png">
